.github/workflows/check-stage.yaml

```
@@ -0,0 +1,26 @@
+        branches:
+            - release/**
+
+jobs:
+    check-stage:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout repo
+              uses: actions/checkout@v4
+              with:
+                fetch-depth: 0 
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 18 but found 16

(indentation)